packages/api/lambdas/create-reconciliation-report.js

```diff
@@ -853,7 +857,8 @@ async function processRequest(params) {
       // reportType is in ['Inventory', 'Granule Not Found']
       await createReconciliationReport(recReportParams);
     }
-    await reconciliationReportModel.updateStatus({ name: reportRecord.name }, 'Generated');
+    apiRecord = await reconciliationReportModel.updateStatus({ name: reportRecord.name }, 'Generated');
+    await indexReconciliationReport(esClient, { ...apiRecord, status: 'Generated' }, process.env.ES_INDEX);
```

Comment:
Should we put it in a Promise.all?

Reply:
The created/updated record from reconciliationReportModel call is used for adding to es, since the record has correct created/updated value.